[2025-01-16 04:02:44,754] ERROR in app: Error getting container details for chatgpt_app1_backend: Command '['docker', 'inspect', 'chatgpt_app1_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:44,803] ERROR in app: Error getting container details for chatgpt_app1_frontend: Command '['docker', 'inspect', 'chatgpt_app1_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:44,849] ERROR in app: Error getting container details for chatgpt_app2_backend: Command '['docker', 'inspect', 'chatgpt_app2_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:44,899] ERROR in app: Error getting container details for chatgpt_app2_frontend: Command '['docker', 'inspect', 'chatgpt_app2_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:44,948] ERROR in app: Error getting container details for chatgpt_app3_backend: Command '['docker', 'inspect', 'chatgpt_app3_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:44,995] ERROR in app: Error getting container details for chatgpt_app3_frontend: Command '['docker', 'inspect', 'chatgpt_app3_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,041] ERROR in app: Error getting container details for chatgpt_app4_backend: Command '['docker', 'inspect', 'chatgpt_app4_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,086] ERROR in app: Error getting container details for chatgpt_app4_frontend: Command '['docker', 'inspect', 'chatgpt_app4_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,131] ERROR in app: Error getting container details for chatgpt_app5_backend: Command '['docker', 'inspect', 'chatgpt_app5_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,179] ERROR in app: Error getting container details for chatgpt_app5_frontend: Command '['docker', 'inspect', 'chatgpt_app5_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,226] ERROR in app: Error getting container details for chatgpt_app6_backend: Command '['docker', 'inspect', 'chatgpt_app6_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,276] ERROR in app: Error getting container details for chatgpt_app6_frontend: Command '['docker', 'inspect', 'chatgpt_app6_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,322] ERROR in app: Error getting container details for chatgpt_app7_backend: Command '['docker', 'inspect', 'chatgpt_app7_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,367] ERROR in app: Error getting container details for chatgpt_app7_frontend: Command '['docker', 'inspect', 'chatgpt_app7_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,413] ERROR in app: Error getting container details for chatgpt_app8_backend: Command '['docker', 'inspect', 'chatgpt_app8_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,457] ERROR in app: Error getting container details for chatgpt_app8_frontend: Command '['docker', 'inspect', 'chatgpt_app8_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,505] ERROR in app: Error getting container details for chatgpt_app9_backend: Command '['docker', 'inspect', 'chatgpt_app9_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,540] ERROR in app: Error getting container details for chatgpt_app1_backend: Command '['docker', 'inspect', 'chatgpt_app1_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,558] ERROR in app: Error getting container details for chatgpt_app9_frontend: Command '['docker', 'inspect', 'chatgpt_app9_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,586] ERROR in app: Error getting container details for chatgpt_app1_frontend: Command '['docker', 'inspect', 'chatgpt_app1_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,605] ERROR in app: Error getting container details for chatgpt_app10_backend: Command '['docker', 'inspect', 'chatgpt_app10_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,650] ERROR in app: Error getting container details for chatgpt_app2_backend: Command '['docker', 'inspect', 'chatgpt_app2_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,669] ERROR in app: Error getting container details for chatgpt_app10_frontend: Command '['docker', 'inspect', 'chatgpt_app10_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,696] ERROR in app: Error getting container details for chatgpt_app2_frontend: Command '['docker', 'inspect', 'chatgpt_app2_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,719] ERROR in app: Error getting container details for chatgpt_app11_backend: Command '['docker', 'inspect', 'chatgpt_app11_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,747] ERROR in app: Error getting container details for chatgpt_app3_backend: Command '['docker', 'inspect', 'chatgpt_app3_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,769] ERROR in app: Error getting container details for chatgpt_app11_frontend: Command '['docker', 'inspect', 'chatgpt_app11_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,798] ERROR in app: Error getting container details for chatgpt_app3_frontend: Command '['docker', 'inspect', 'chatgpt_app3_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,818] ERROR in app: Error getting container details for chatgpt_app12_backend: Command '['docker', 'inspect', 'chatgpt_app12_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,847] ERROR in app: Error getting container details for chatgpt_app4_backend: Command '['docker', 'inspect', 'chatgpt_app4_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,870] ERROR in app: Error getting container details for chatgpt_app12_frontend: Command '['docker', 'inspect', 'chatgpt_app12_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,899] ERROR in app: Error getting container details for chatgpt_app4_frontend: Command '['docker', 'inspect', 'chatgpt_app4_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,921] ERROR in app: Error getting container details for chatgpt_app13_backend: Command '['docker', 'inspect', 'chatgpt_app13_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,951] ERROR in app: Error getting container details for chatgpt_app5_backend: Command '['docker', 'inspect', 'chatgpt_app5_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:45,978] ERROR in app: Error getting container details for chatgpt_app13_frontend: Command '['docker', 'inspect', 'chatgpt_app13_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,001] ERROR in app: Error getting container details for chatgpt_app5_frontend: Command '['docker', 'inspect', 'chatgpt_app5_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,029] ERROR in app: Error getting container details for chatgpt_app14_backend: Command '['docker', 'inspect', 'chatgpt_app14_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,053] ERROR in app: Error getting container details for chatgpt_app6_backend: Command '['docker', 'inspect', 'chatgpt_app6_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,078] ERROR in app: Error getting container details for chatgpt_app14_frontend: Command '['docker', 'inspect', 'chatgpt_app14_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,102] ERROR in app: Error getting container details for chatgpt_app6_frontend: Command '['docker', 'inspect', 'chatgpt_app6_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,125] ERROR in app: Error getting container details for chatgpt_app15_backend: Command '['docker', 'inspect', 'chatgpt_app15_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,157] ERROR in app: Error getting container details for chatgpt_app7_backend: Command '['docker', 'inspect', 'chatgpt_app7_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,194] ERROR in app: Error getting container details for chatgpt_app15_frontend: Command '['docker', 'inspect', 'chatgpt_app15_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,222] ERROR in app: Error getting container details for chatgpt_app7_frontend: Command '['docker', 'inspect', 'chatgpt_app7_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,254] ERROR in app: Error getting container details for chatgpt_app16_backend: Command '['docker', 'inspect', 'chatgpt_app16_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,277] ERROR in app: Error getting container details for chatgpt_app8_backend: Command '['docker', 'inspect', 'chatgpt_app8_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,308] ERROR in app: Error getting container details for chatgpt_app16_frontend: Command '['docker', 'inspect', 'chatgpt_app16_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,328] ERROR in app: Error getting container details for chatgpt_app8_frontend: Command '['docker', 'inspect', 'chatgpt_app8_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,359] ERROR in app: Error getting container details for chatgpt_app17_backend: Command '['docker', 'inspect', 'chatgpt_app17_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,381] ERROR in app: Error getting container details for chatgpt_app9_backend: Command '['docker', 'inspect', 'chatgpt_app9_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,411] ERROR in app: Error getting container details for chatgpt_app17_frontend: Command '['docker', 'inspect', 'chatgpt_app17_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,434] ERROR in app: Error getting container details for chatgpt_app9_frontend: Command '['docker', 'inspect', 'chatgpt_app9_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,461] ERROR in app: Error getting container details for chatgpt_app18_backend: Command '['docker', 'inspect', 'chatgpt_app18_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,481] ERROR in app: Error getting container details for chatgpt_app10_backend: Command '['docker', 'inspect', 'chatgpt_app10_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,512] ERROR in app: Error getting container details for chatgpt_app18_frontend: Command '['docker', 'inspect', 'chatgpt_app18_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,532] ERROR in app: Error getting container details for chatgpt_app10_frontend: Command '['docker', 'inspect', 'chatgpt_app10_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,562] ERROR in app: Error getting container details for chatgpt_app19_backend: Command '['docker', 'inspect', 'chatgpt_app19_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,585] ERROR in app: Error getting container details for chatgpt_app11_backend: Command '['docker', 'inspect', 'chatgpt_app11_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,612] ERROR in app: Error getting container details for chatgpt_app19_frontend: Command '['docker', 'inspect', 'chatgpt_app19_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,633] ERROR in app: Error getting container details for chatgpt_app11_frontend: Command '['docker', 'inspect', 'chatgpt_app11_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,661] ERROR in app: Error getting container details for chatgpt_app20_backend: Command '['docker', 'inspect', 'chatgpt_app20_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,685] ERROR in app: Error getting container details for chatgpt_app12_backend: Command '['docker', 'inspect', 'chatgpt_app12_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,710] ERROR in app: Error getting container details for chatgpt_app20_frontend: Command '['docker', 'inspect', 'chatgpt_app20_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,735] ERROR in app: Error getting container details for chatgpt_app12_frontend: Command '['docker', 'inspect', 'chatgpt_app12_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,757] ERROR in app: Error getting container details for claudesonnet_app1_backend: Command '['docker', 'inspect', 'claudesonnet_app1_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,782] ERROR in app: Error getting container details for chatgpt_app13_backend: Command '['docker', 'inspect', 'chatgpt_app13_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,806] ERROR in app: Error getting container details for claudesonnet_app1_frontend: Command '['docker', 'inspect', 'claudesonnet_app1_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,831] ERROR in app: Error getting container details for chatgpt_app13_frontend: Command '['docker', 'inspect', 'chatgpt_app13_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,858] ERROR in app: Error getting container details for claudesonnet_app2_backend: Command '['docker', 'inspect', 'claudesonnet_app2_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,888] ERROR in app: Error getting container details for chatgpt_app14_backend: Command '['docker', 'inspect', 'chatgpt_app14_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,918] ERROR in app: Error getting container details for claudesonnet_app2_frontend: Command '['docker', 'inspect', 'claudesonnet_app2_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,951] ERROR in app: Error getting container details for chatgpt_app14_frontend: Command '['docker', 'inspect', 'chatgpt_app14_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:46,970] ERROR in app: Error getting container details for claudesonnet_app3_backend: Command '['docker', 'inspect', 'claudesonnet_app3_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,018] ERROR in app: Error getting container details for chatgpt_app15_backend: Command '['docker', 'inspect', 'chatgpt_app15_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,025] ERROR in app: Error getting container details for claudesonnet_app3_frontend: Command '['docker', 'inspect', 'claudesonnet_app3_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,065] ERROR in app: Error getting container details for chatgpt_app15_frontend: Command '['docker', 'inspect', 'chatgpt_app15_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,072] ERROR in app: Error getting container details for claudesonnet_app4_backend: Command '['docker', 'inspect', 'claudesonnet_app4_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,111] ERROR in app: Error getting container details for chatgpt_app16_backend: Command '['docker', 'inspect', 'chatgpt_app16_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,118] ERROR in app: Error getting container details for claudesonnet_app4_frontend: Command '['docker', 'inspect', 'claudesonnet_app4_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,157] ERROR in app: Error getting container details for chatgpt_app16_frontend: Command '['docker', 'inspect', 'chatgpt_app16_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,165] ERROR in app: Error getting container details for claudesonnet_app5_backend: Command '['docker', 'inspect', 'claudesonnet_app5_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,205] ERROR in app: Error getting container details for chatgpt_app17_backend: Command '['docker', 'inspect', 'chatgpt_app17_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,214] ERROR in app: Error getting container details for claudesonnet_app5_frontend: Command '['docker', 'inspect', 'claudesonnet_app5_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,253] ERROR in app: Error getting container details for chatgpt_app17_frontend: Command '['docker', 'inspect', 'chatgpt_app17_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,260] ERROR in app: Error getting container details for claudesonnet_app6_backend: Command '['docker', 'inspect', 'claudesonnet_app6_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,325] ERROR in app: Error getting container details for chatgpt_app18_backend: Command '['docker', 'inspect', 'chatgpt_app18_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,332] ERROR in app: Error getting container details for claudesonnet_app6_frontend: Command '['docker', 'inspect', 'claudesonnet_app6_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,374] ERROR in app: Error getting container details for chatgpt_app18_frontend: Command '['docker', 'inspect', 'chatgpt_app18_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,374] ERROR in app: Error getting container details for claudesonnet_app7_backend: Command '['docker', 'inspect', 'claudesonnet_app7_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,421] ERROR in app: Error getting container details for claudesonnet_app7_frontend: Command '['docker', 'inspect', 'claudesonnet_app7_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,421] ERROR in app: Error getting container details for chatgpt_app19_backend: Command '['docker', 'inspect', 'chatgpt_app19_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,468] ERROR in app: Error getting container details for chatgpt_app19_frontend: Command '['docker', 'inspect', 'chatgpt_app19_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,480] ERROR in app: Error getting container details for claudesonnet_app8_backend: Command '['docker', 'inspect', 'claudesonnet_app8_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,516] ERROR in app: Error getting container details for chatgpt_app20_backend: Command '['docker', 'inspect', 'chatgpt_app20_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,531] ERROR in app: Error getting container details for claudesonnet_app8_frontend: Command '['docker', 'inspect', 'claudesonnet_app8_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,563] ERROR in app: Error getting container details for chatgpt_app20_frontend: Command '['docker', 'inspect', 'chatgpt_app20_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,578] ERROR in app: Error getting container details for claudesonnet_app9_backend: Command '['docker', 'inspect', 'claudesonnet_app9_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,610] ERROR in app: Error getting container details for claudesonnet_app1_backend: Command '['docker', 'inspect', 'claudesonnet_app1_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,627] ERROR in app: Error getting container details for claudesonnet_app9_frontend: Command '['docker', 'inspect', 'claudesonnet_app9_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,657] ERROR in app: Error getting container details for claudesonnet_app1_frontend: Command '['docker', 'inspect', 'claudesonnet_app1_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,674] ERROR in app: Error getting container details for claudesonnet_app10_backend: Command '['docker', 'inspect', 'claudesonnet_app10_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,707] ERROR in app: Error getting container details for claudesonnet_app2_backend: Command '['docker', 'inspect', 'claudesonnet_app2_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,720] ERROR in app: Error getting container details for claudesonnet_app10_frontend: Command '['docker', 'inspect', 'claudesonnet_app10_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,756] ERROR in app: Error getting container details for claudesonnet_app2_frontend: Command '['docker', 'inspect', 'claudesonnet_app2_frontend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,771] ERROR in app: Error getting container details for claudesonnet_app11_backend: Command '['docker', 'inspect', 'claudesonnet_app11_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,805] ERROR in app: Error getting container details for claudesonnet_app3_backend: Command '['docker', 'inspect', 'claudesonnet_app3_backend']' returned non-zero exit status 1.
[2025-01-16 04:02:47,819] ERROR in app: Error getting container details for claudesonnet_app11_frontend: Command '['docker', 'inspect', 'claudesonnet_app11_frontend']' returned non-zero exit status 1.
[2025-01-16 04:05:32,969] ERROR: Error loading index: 'dict object' has no attribute 'docker'
